test/integration/short/prepare-simulator-tests.js

```diff
@@ -0,0 +1,142 @@
+'use strict';
+var assert = require('assert');
+var util = require('util');
+
+var Client = require('../../../lib/client');
+var types = require('../../../lib/types/index');
+var utils = require('../../../lib/utils');
+var helper = require('../../test-helper');
+var reconnection = require('../../../lib/policies/reconnection');
+var simulacronHelper = require('../simulacron-helper');
+
+describe('Client #prepare', function () {
+  this.timeout(20000);
+  describe('Preparing statements on nodes behavior', function () {
+    var nodes = "5";
+    var clusterName = "simulacronTest";
+    var cassandraVersion = "3.10";
+    var sCluster = null;
+    var client = null;
+    var query = util.format('SELECT * FROM ks.table1 WHERE id1 = ?');
+    before(function (done) {
+      simulacronHelper.start(done);
+    });
+    beforeEach(function (done) {
+      utils.series(
+        [
+          function startCluster(next) {
+            sCluster = new simulacronHelper.SimulacronCluster();
+            sCluster.start(nodes, cassandraVersion, "", clusterName, true, 1, next);
+          },
+          function connectCluster(next) {
+            client = new Client({
+              contactPoints: [sCluster.dcs[0].nodes[0].address],
```

Comment:
A convenience function for getting all the addresses would be nice.   i.e. `getContactPoints` which simply returns the addresses of all nodes in dc0.